README.md

```diff
@@ -13,8 +13,8 @@ Here is how you should install:
 git clone https://github.com/cta-observatory/nectarchain.git
 cd nectarchain
 conda env create --name cta --file environment.yml
-source activate cta
-pip install https://github.com/cta-observatory/ctapipe/archive/master.tar.gz
+conda activate cta
+conda install -c conda-forge ctapipe
```

Comment:
To be discussed with @FrancaCassol , but we decided at some point that `nectarchain` should always depend on the master of `ctatpipe`, not on tagged versions.